required/latex-lab/latex-lab-float.dtx

```diff
@@ -383,7 +462,8 @@
 %    \end{macrocode}
 % This starts the structure for the float.
 %    \begin{macrocode}
-  \@nameuse{@@_float_begin:}%
+  \csname @@_float_init:\endcsname
```

Comment:
Why this change?

Reply:
The `\@@_float_begin:` command has been replaced by `\UseTaggingSocket{float/begin}`. The `\@@_float_init:`  is new, it takes care of the bits that must be initialized outside a tagging socket (like a counter for link targets). Why I used `\csname/\endcsname` instead of `\@nameuse`  I don't know ;-). Do we have a preference here?

Reply:
yes, probably. But I'm not sure yet if the command itself should be public, or if there should be a wrapper socket or similar. So I would keep that on hold for now.

Reply:
ok, but then I would add a TODO  there